src/Sylius/Bundle/AdminApiBundle/Resources/config/routing/channel.yml

```diff
@@ -6,7 +6,6 @@ sylius_admin_api_channel:
         identifier: code
         alias: sylius.channel
         section: admin_api
-        only: ['show', 'index']
```

Comment:
The update action is not tested, so `except: ['update']` is needed or a few more tests